airflow/cli/commands/celery_command.py

```diff
@@ -125,7 +125,10 @@ def worker(args):
     }
 
     if conf.has_option("celery", "pool"):
-        options["pool"] = conf.get("celery", "pool")
+        pool = conf.get("celery", "pool")
+        options["pool"] = pool
+        from celery import maybe_patch_concurrency
```

Comment:
Let's move this import to the top, we are importing whole celery anyway

Reply:
Sounds good.